lib/media/content_workarounds.js

```diff
@@ -381,7 +381,7 @@ shaka.media.ContentWorkarounds = class {
           ancestorBoxes.push({start: box.start, size: box.size});
           const stsdBoxView = shaka.util.BufferUtils.toDataView(
               initSegment, box.start);
-          for (let i=0; i<box.size; i++) {
+          for (let i = 0; i < box.size -3; i++) {
```

Comment:
Please leave a comment that "size - 3" is because we immediately read a uint32.

Reply:
Done!